pkg/oonimkall/session_integration_test.go

```diff
@@ -419,6 +419,67 @@ func TestCheckInNoParams(t *testing.T) {
 	}
 }
 
+func TestFetchURLListSucc(t *testing.T) {
+	sess, err := NewSession()
+	if err != nil {
+		t.Fatal(err)
+	}
+	ctx := sess.NewContext()
+	config := oonimkall.URLListConfig{
+		Limit: 10,
+	}
+	config.AddCategory("NEWS")
+	config.AddCategory("CULTR")
+	result, err := sess.FetchURLList(ctx, &config)
+
+	if err != nil {
+		t.Fatalf("unexpected error: %+v", err)
+	}
+	if result == nil || result.Results == nil {
+		t.Fatal("got nil result")
+	}
+	if len(result.Results) < 1 {
```

Comment:
`len(result.Results)` is zero when `result.Results` is `nil`, so you can write more compact code above.

Reply:
Is there a way to differentiate between nil and empty array? I don't know how you want the code to behave

Reply:
In this case an empty array or a nil array is wrong, so I don't think we need to distinguish.